_translator-files/po/fr/1-index.po

```diff
@@ -6,14 +6,16 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"PO-Revision-Date: 2022-08-04 01:33+0200\n"
-"Last-Translator: \n"
-"Language-Team: \n"
+"PO-Revision-Date: 2022-08-15 15:17+0000\n"
+"Last-Translator: Julien Taverna <jujudusud@gmail.com>\n"
```

Comment:
@jujudusud @ignotus666 Is it ok to have E-Mails here?
I'd rather not like to have mine in here

Reply:
I don't know if it is a good or a bad things.

Reply:
I see it as a bit sub optimal (spam etc). My E-Mail can be found in my GH profile in encoded form, but we don't know if all contributors want their Mail being displayed here.